packages/cli/src/utils/exposeLoader.ts

```diff
@@ -44,9 +44,12 @@ export const pitch = function(remainingRequest) {
    * - https://github.com/webpack-contrib/expose-loader/issues/49
    */
   // eslint-disable-next-line no-underscore-dangle
-  this._module.userRequest = `${this._module.userRequest}-exposed`;
-  return (
-    `${accesorString(this.query.substr(1))} = ` +
-    `require(${JSON.stringify(`-!${newRequestPath}`)});`
-  );
+  const originalUserRequest = this._module.userRequest;
+  // eslint-disable-next-line no-underscore-dangle
+  this._module.userRequest = `${originalUserRequest}-exposed`;
+  const symbol =
```

Comment:
can you ELI5 what we are doing here?

Reply:
This is mostly copy paste from `expose-loader` library.
`symbol` comes from the `options` in webpack.
it can either be a function that receives the file path or it's a string.

This is to define the name for the variable that is exposed in `window`.